[invalid_capabilities.py]
  [test_invalid_values[timeouts-value48-body0\]]
    expected: FAIL

  [test_invalid_values[timeouts-value48-body1\]]
    expected: FAIL

